src/components/badges/ListTitle.tsx

```diff
@@ -8,15 +8,23 @@ import useContractsOwned from 'hooks/useContractsOwned'
 import useProofsAvailableToMint from 'hooks/useProofsAvailableToMint'
 
 function ListTitleSuspended() {
-  const { derivativeContracts = [] } = useSnapshot(SealCredStore)
+  const { derivativeContracts } = useSnapshot(SealCredStore)
   const proofsAvailableToMint = useProofsAvailableToMint()
-  const contractsOwned = useContractsOwned(GoerliContractsStore)
+  const contractsOwned = useContractsOwned(BadgesContractsStore)
 
-  const ownedDerivativeContracts = derivativeContracts.filter(
-    (contractAddress) => contractsOwned.includes(contractAddress)
+  const ownedDerivativeContracts = Object.values(derivativeContracts).reduce(
+    (chain, contracts) =>
+      chain.concat(
+        contracts.filter((contractAddress) =>
+          contractsOwned.includes(contractAddress)
+        )
+      ),
+    []
   )
 
-  const hasUnminted = proofsAvailableToMint.length > 0
+  const hasUnminted = Object.values(proofsAvailableToMint).some(
```

Comment:
You're doing this operation in `badges/List.tsx/L35`, maybe it's wors extracting this operation into `helpers`? Something like `checkObjectValuesHasSome`

Reply:
If it's not easy to do, let's leave it like this